### ISSUE TITLE
Edge intersection style in nodeless mode

### ISSUE THREAD

**Issue by dan-knight** [uclahs-cds/package-CancerEvolutionVisualization#118]
Without nodes, the intersection between edge line segments becomes visible. How should we handle this detail?

<img width="177" alt="Screenshot 2024-02-13 at 12 24 03 PM" src="https://github.com/uclahs-cds/package-CancerEvolutionVisualization/assets/56950335/4aef6491-8efc-4279-9bdb-d719464cde1a">

**Comment by dan-knight** [comment]
Other considerations:

**Varying line segment thickness**
<img width="204" alt="Screenshot 2024-02-13 at 12 43 56 PM" src="https://github.com/uclahs-cds/package-CancerEvolutionVisualization/assets/56950335/5311015f-92fd-4289-b9f4-930e0287fc53">

**Varying line type**
<img width="187" alt="Screenshot 2024-02-13 at 12 45 35 PM" src="https://github.com/uclahs-cds/package-CancerEvolutionVisualization/assets/56950335/0f51d2b0-0081-410e-9861-3400834a113e">

**Comment by dan-knight** [comment]
# Possible Solutions

**Parent on Top Layer**
Grobs are stacked in the order that they are created, so the later line segments are rendered on top of the earlier ones. By reversing this, the parent will be rendered on top of its children. This results in an (almost) node at the tip of the parent line segment.
<img width="202" alt="Screenshot 2024-02-13 at 1 07 39 PM" src="https://github.com/uclahs-cds/package-CancerEvolutionVisualization/assets/56950335/e722dfc8-a25d-43e1-80aa-3e373a2ee98c">

**Comment by whelena** [comment]
I might be missing the point of this issue, but what specifically are we considering here? Are you implying that there is a difference in how we would interpret the plot depending if the parent edge vs. children edge is on the front layer?

**Comment by dan-knight** [comment]
I don't think that the visual style was considered very much in the original design, since the node circles would hide this part of the plot. Now, nodeless mode will be more relevant when combined with dendrogram mode. Are there any major issues you can see here? General ugliness? It may just be the nature of these plots - something we leave for users to resolve given the amount of control the package allows.

**Comment by pboutros** [comment]
I think... at least for now, not a high priority issue.  I view this as a graphical nicety rather than an error, and I think worth keeping this as an open feature request "find ways to improve visualization of segment boundaries in nodeless mode" kinda thing, but not worth stressing about right now as this is exposing features to users and allowing them to decide on optimal application.

**Comment by dan-knight** [comment]
> I view this as a graphical nicety rather than an error

Sounds good. I don't think there's anything obvious that we should enforce as a default here. Certainly nothing we should enforce as a constraint. Thanks for the input everyone!
